src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx

```diff
@@ -27,7 +31,7 @@ Requirements:
 * Meet [PHP agent requirements](/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements)
 * PHP agent version 9.2 or higher
 
-To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841).
+To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841). Using a separate container for the daemon is recommended if there will be short-lived application containers.
```

Comment:
suggestion: `If using short-lived application containers, it is recommended to use a separate container for the PHP Agent daemon.`